components/demo/demo-snippet.js

```diff
@@ -122,7 +122,7 @@ class DemoSnippet extends LitElement {
 							nodes[i].setAttribute(name, value);
 						}
 					}
-					if (nodes[i].shadowRoot) {
+					if (applyToShadowRoot && nodes[i].shadowRoot) {
```

Comment:
Slight change here as we don't want it to travel into shadow roots like the RTL one does and turn skeletons on -- a web component is responsible for passing down `skeleton` to any components inside itself.